BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/TelemetryClientTest.cs

```diff
@@ -447,62 +445,61 @@ public void TrackMetricWithProperties()
                             break;
                         }
                     }
-                    Assert.IsTrue(hasProperty, "Metric should have property1 tag");
+                    Assert.True(hasProperty, "Metric should have property1 tag");
                     break;
                 }
             }
-            Assert.IsTrue(hasData, "Metric should have recorded data");
+            Assert.True(hasData, "Metric should have recorded data");
         }
 
-        [TestMethod]
+        [Fact]
         public void TrackMetricWithNullPropertiesDoesNotThrow()
         {
             this.telemetryClient.TrackMetric("TestMetric", 4.2, null);
             this.telemetryClient.Flush();
-            Assert.IsTrue(this.metricItems.Count > 0, "At least one metric should be collected");
+            Assert.True(this.metricItems.Count > 0, "At least one metric should be collected");
             var metric = this.metricItems.FirstOrDefault(m => m.Name == "TestMetric");
-            Assert.IsNotNull(metric, "TestMetric should be collected");
+            Assert.NotNull(metric);
         }
 
-        [TestMethod]
+        [Fact]
         public void GetMetricWithZeroDimensions()
         {
             var metric = this.telemetryClient.GetMetric("TestMetric");
-            Assert.IsNotNull(metric);
+            Assert.NotNull(metric);
             
             metric.TrackValue(10.0);
             metric.TrackValue(20.0);
             
             this.telemetryClient.Flush();
-            Assert.IsTrue(this.metricItems.Count > 0, "At least one metric should be collected");
+            Assert.True(this.metricItems.Count > 0);
```

Comment:
nit: Why did most `Assert.True` keep the custom message and this one removed it? Other asserts like `Assert.NotNull` and `Assert.Equal` should also the the custom message but most removed it.